recipes-core/watchdog/files/99-watchdog.conf

```diff
@@ -0,0 +1,3 @@
+[Manager]
+RuntimeWatchdogSec=60s
```

Comment:
Hm, shouldn't this be configurable, e.g., by a template variable?

Reply:
After my testing:  The watchdog driver doesn't support updating the timeout
after start. Changing this value will trigger periodic reboots.

Reply:
See https://github.com/siemens/meta-iot2050/blob/master/recipes-kernel/linux/files/patches-5.10/0104-watchdog-rti-wdt-Provide-set_timeout-handler-to-make.patch. The actual tunable is in the u-boot script.

Reply:
But, true, both must be in sync with that temporary patch above. Upstream systemd went for a different approach: https://github.com/systemd/systemd/pull/20787.

Reply:
I will add a comment - to the file. @jan-kiszka Does setting the timeout via kernel command line work?

Reply:
Nope, the timeout is defined in this hardware when starting it. Thus it is defined by U-Boot.